apidoc/Titanium/UI/ListView.yml

```diff
@@ -1058,40 +1058,6 @@ properties:
     osver: {ios: {min: "7.0"}}
     platforms: [iphone, ipad]
 
-  - name: resultsBackgroundColor
-    summary: The background color of the search results (iOS-only).
-    description: |
-        For information about color values, see the "Colors" section of <Titanium.UI>. 
-        Note: A transparent background-color is not officially supported by Apple to
-        prevent that the list of results overlaps with the list view below it.  
-    type: String
-    default: undefined (behaves as white) 
-    since: 6.1.0
-    platforms: [iphone, ipad]
-    availability: creation
-
-  - name: resultsSeparatorColor
-    summary: | 
-        Separator line color between rows inside search results, 
-        as a color name or hex triplet (iOS-only).
-    description: |
-        To make the line invisible, set this property to `transparent`, or the same value as the 
-        [backgroundColor](Titanium.UI.ListView.backgroundColor) property. 
-        For information about color values, see the "Colors" section of <Titanium.UI>. 
-    type: String
-    default: undefined (behaves as gray) 
-    since: 6.1.0
-    platforms: [iphone, ipad]
-    availability: creation
-
-  - name: resultsSeparatorStyle
-    summary: The separator style of the search results (iOS-only).
-    type: Number
-    constants: Titanium.UI.iPhone.ListViewSeparatorStyle.*
-    since: 6.1.0
-    platforms: [iphone, ipad]
-    availability: creation
```

Comment:
Agree, but we need to state that, by using:

```yml
removed: 
    since: "6.2.0"
    notes: |
        The styling of the search table-view is determined by it's table-view, so there
        is no additional styling required.
```